cubed/tests/test_array_api.py

```diff
@@ -93,6 +93,13 @@ def test_asarray_from_array(spec):
     assert_array_equal(b.compute(), np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]]))
 
 
+def test_from_array(spec):
```

Comment:
This test belongs in `test_core.py` since the function it is testing (`from_array`) is not in the array API.